docs/state.md

```diff
@@ -279,16 +279,16 @@ AppStore
     audible: boolean, // is audio playing (muted or not)
     canGoBack: boolean, // the tab can be navigated back
     canGoForward: boolean, // the tab can be navigated forward
-    muted: boolean, // is the tab muted
-    windowId: number, // the windowId that contains the tab
     messageBoxDetail: { // fields used if showing a message box for a tab
       message: string,
```

Comment:
There is a comment showing some which are session and some which are persistent. We can alphabetize but then remove that label, but I don't see value with that